bower.json

```diff
@@ -25,7 +25,7 @@
     "test"
   ],
   "dependencies": {
-    "ember": ">= 2.7.3 <= 3.0.0"
+    "ember": "2.7.3"
```

Comment:
was this causing a problem?  I had done this so that we didn't have to specify a resolution in dashboard for our ember version.

(although looking at this again I probably should have been `< 3.0.0`